### ISSUE TITLE
Chase constiution

### ISSUE THREAD

**Issue by PatReynolds** [FreeUKGen/Coordination#810]

**Comment by PatReynolds** [comment]
Filled in another Charity Commission form 17/7/2019, now have reference number: 1523292

**Comment by PatReynolds** [comment]
Agreed - and returned final document to CC 29/08/2019.  This is the pdf in /governance "version 3" and renamed to "Free UK Genealogy Foundation CIOConstitution.pdf" in Craft, but it is still showing version 2....

Could you check on your return, please, @DeniseColbert

**Comment by PatReynolds** [comment]
Checked 17 Jan 2020, still old objects.

**Comment by DeniseColbert** [comment]
Sorry, I missed this. What's different between version 2 and version 3? The one showing on the website and version 3 in the drive have the same Adopted and Amended dates. I can change the Craft PDF to the drive's Version 3 file, but need to know the above to be able to check it worked.

**Comment by DeniseColbert** [comment]
Should be fixed now (sorry I missed it). There were '-' in the new filename and none in the old one. It's a charming (!) idiosyncrasy of Craft that files deleted from the CMS are not really deleted and still shown.

**Comment by PatReynolds** [comment]
Well actually, I was needing to chase the Charity Comission who have still got the old, pre-CIO version ...

**Comment by PatReynolds** [comment]
Found email thread: they have agreed the changes, but wanted some paperwork, which I sent in 2019.  I have resent, and cc'd you into the email.
